File: Programmers/Level2/튜플/튜플.py
def solution(s):
    answer = []
    lst = sorted([i.split(',') for i in s[2:-2].split('},{')], key=len)

    for l in lst:
        for s in l:
            if int(s) not in answer:
                answer.append(int(s))
                continue
    return answer

File: Programmers/Level2/튜플/튜플.js
function solution(s) {
    let answer = [];
    arr = s
        .slice(2, -2)
        .split("},{")
        .map((e) => e.split(","))
        .sort((a, b) => a.length - b.length);

    for (let a of arr) {
        for (let s of a) {
            if (!answer.includes(parseInt(s))) {
                answer.push(parseInt(s));
                continue;
            }
        }
    }
    return answer;
}
